mappings/net/minecraft/item/tool/ItemTool.mapping

```diff
@@ -1,5 +1,35 @@
 CLASS none/adr net/minecraft/item/tool/ItemTool
+	FIELD a efficiency F
+	FIELD b weaponDamage F
+	FIELD c swingSpeed F
 	FIELD d material Lnone/aeo$a;
+	FIELD e EFFECTIVE_BLOCKS Ljava/util/Set;
 	METHOD <init> (FFLnone/aeo$a;Ljava/util/Set;)V
-		ARG 2 toolMaterial
+		ARG 0 damageModifier
+		ARG 1 swingSpeed
+		ARG 2 material
+		ARG 3 effectiveBlocks
+	METHOD <init> (Lnone/aeo$a;Ljava/util/Set;)V
+		ARG 0 material
+		ARG 1 effectiveBlocks
+	METHOD a canRepair (Lnone/aeq;Lnone/aeq;)Z
+		ARG 0 tool
+		ARG 1 repairItem
+	METHOD a onBlockBroken (Lnone/aeq;Lnone/aiu;Lnone/asj;Lnone/cn;Lnone/sn;)Z
+		ARG 0 stack
+		ARG 1 world
+		ARG 2 state
+		ARG 3 pos
+		ARG 4 user
+	METHOD a getMiningStrength (Lnone/aeq;Lnone/asj;)F
+		ARG 0 stack
+		ARG 1 state
+	METHOD a onEntityDamaged (Lnone/aeq;Lnone/sn;Lnone/sn;)Z
+		ARG 0 stack
+		ARG 1 target
+		ARG 2 user
+	METHOD a getAttributeModifiers (Lnone/sj;)Lcom/google/common/collect/Multimap;
+		ARG 0 slotType
+	METHOD c getEnchantability ()I
 	METHOD g getMaterial ()Lnone/aeo$a;
+	METHOD h getMaterial ()Ljava/lang/String;
```

Comment:
This is a typo on my end. The one which returns String should be getMaterialName. Will fix.